source/lifecycle-management/transition-objects-to-s3.rst

```diff
@@ -167,6 +176,11 @@ The example above uses the following arguments:
      - The optional bucket prefix within which MinIO transitions objects.
        Omit this argument to transition objects to the bucket root.
 
+       MinIO strongly recommends specifying a prefix which provides clear
```

Comment:
Added this section because it occurred to me that, if I am dumping multiple deployments into the same remote backend, I have no idea what is safe to delete and what isn't. 

This way at least you have per-deployment isolation on the remote, so if one deployment goes down, you can purge that and not have to figure out how to hand-pick through everything.